library/src/scripts/components/forms/select/SearchBar.tsx

```diff
@@ -39,6 +41,9 @@ interface IProps extends IOptionalComponentID {
     isLoading?: boolean;
     onSearch: () => void;
     optionComponent?: React.ComponentType<OptionProps<any>>;
+    getRef?: any;
```

Comment:
That ref is for the "compact search" for the header... Why don't we meet next sprint about that component. It might be best to refactor a bit next sprint.